designs/otis/src/shortsleeve.mjs

```diff
@@ -0,0 +1,142 @@
+import { pluginCutlist } from '@freesewing/plugin-cutlist'
+import { pluginBundle } from '@freesewing/plugin-bundle'
+import { back } from './back.mjs'
+
+function draftShortsleeve({
+  options,
+  Point,
+  Path,
+  points,
+  paths,
+  Snippet,
+  snippets,
+  complete,
+  sa,
+  paperless,
+  store,
+  macro,
+  utils,
+  part,
+}) {
+  const hem = options.hem
+  const waist = store.get('waist')
+  const ease = store.get('ease')
+  const sizeFactor = store.get('sizeFactor')
+  const armhole = store.get('armhole')
+  store.set('hem', 117 * (ease + 1) * sizeFactor * hem)
+
+  points.p0 = new Point(0, 0)
+  points.p0Cp2 = points.p0.shift(0, 84.746 * (ease + 1) * sizeFactor)
+  points.p1 = points.p0.shift(270, 117 * (ease + 1) * sizeFactor)
+  points.p2 = points.p1.shift(0, 121 * (ease + 1) * sizeFactor)
+  points.p3 = points.p0.shift(331.965149, 145.468319 * (ease + 1) * sizeFactor)
+  points.p3Cp1 = points.p3.shift(180, 69.454559985 * (ease + 1) * sizeFactor)
+
+  let diff = 10
+  let iter = 1
+  do {
+    paths.armhole = new Path().move(points.p3).curve(points.p3Cp1, points.p0Cp2, points.p0).hide()
+
+    diff = armhole - paths.armhole.length()
+    if (diff < -1 || diff > 1) {
+      points.p3 = points.p0.shift(331.965149, 145.468319 * (ease + 1) * sizeFactor + diff)
+    }
+    iter++
+  } while ((diff < -1 || diff > 1) & (iter < 100))
```

Comment:
This is using the `&` operator which is almost certainly not what you want, because it's a bitwise AND which expects two numbers, but the left part is always going to be a boolean.

You probably want the `&&` operator, which is the logical AND.

Please address this because it's a bug.

Reply:
Well, that's embarrasing!